discover/portal/articles-dxp/140-workflow/02-kaleo-forms/02-assigning-kaleo-forms.markdown

```diff
@@ -72,16 +78,21 @@ over it and clicking the wrench icon.
 
 $$$
 
-If you don't need to change any field settings in the form, click *Save*. On the
-next screen choose your new form by clicking on it. Now the workflow
-definition's StartNode has a form associated with it.
+You have three more fields to create using these steps.
 
-Next create a form named *Spa Order Process--Technician Claim Form* for the
-Technician Claim workflow task. Follow the same procedure used above, this time
-deleting these fields from the full field set:
+1.  *Spa Order Process--Technical Claim Form*
 
-- Approved
-- Managerial Comments
+    For the Technician Claim workflow task. Follow the same procedure used 
```

Comment:
On the screenshots, this Workflow task is named as _Technician_ instead of _Technician Claim_. 🙂